kalkulator.py

```diff
@@ -3,7 +3,13 @@ def get_info():
 def dodaj(a,b):
     wynik = a + b
     return wynik
+    
+ def odejmij(a,b):
+    return a - b
```

Comment:
Moim zdaniem tutaj lepiej rozbić to na dwie linijki wprowadzając zmienną "wynik"